bin/tag-and-deploy-docker-image

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -e
+
+currentDir=$(node -e "console.log(require('path').dirname(require('fs').realpathSync('$BASH_SOURCE')))")
+source $currentDir/helper
+
+help () {
+  log "You MUST defined the following environment variables:"
+  log ""
+  log "<RELEASE_SERVER_HOST>             Hostname of livingdocs/release-server"
+  log "<RELEASE_SERVER_SERVICE_HANDLE>   Service handle of livingdocs/release-server"
+  exit 1
+}
+
+deploy_to_rancher () {
+  curl -i -XPOST -H 'Content-Type:application/json' -d "{\"serviceHandle\":\"$RELEASE_SERVER_SERVICE_HANDLE\",\"dockerImageTag\":\"$1\"}" $RELEASE_SERVER_HOST/deploy;
```

Comment:
Can we use single quotes for the `-d` option so that we don't have to escape them inside the string?